packages/libp2p/test/circuit-relay/relay.node.ts

```diff
@@ -675,6 +675,26 @@ describe('circuit-relay', () => {
       expect(events[1].detail.remotePeer.toString()).to.equal(relay1.peerId.toString())
     })
 
+    // Not sure why this test isn't passing
+    it.skip('should remove the relay event listener when the relay stops', async () => {
```

Comment:
I am not sure why there aren't any listeners being attached to the relay as the logs are saying otherwise, but `listenerCount` is returning 0.